config/commands.js

```diff
@@ -507,6 +507,7 @@ var commands = exports.commands = {
 				continue;
 			}
 
+			if (target.substr(0, 4) === 'gen ' && !isNaN(target.substr(4))) target = target.substr(4);
```

Comment:
Maybe you could change this to <code>if (target.substr(0, 3) === 'gen' && !isNaN(target.substr(3))) target = target.substr(3).trim();</code> so that both <code>/ds gen #</code> and <code>/ds gen#</code> would work.